integrations/extensions/docs/elasticsearch-install-and-setup/how_to_index_pdf_and_office_documents_elasticsearch.md

```diff
@@ -0,0 +1,216 @@
+
+```bash
+curl -X PUT "${ES_URL}/_ingest/pipeline/${ES_PIPELINE_NAME}?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+-H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+{
+  "processors": [
+  {
+    "script": {
+      "source": " String[] envSplit = /((?<!M(r|s|rs)\\.)(?<=\\.) |(?<=\\!) |(?<=\\?) )/.split(ctx['\''content'\'']);\n          ctx['\''passages'\''] = new ArrayList();\n          int i = 0;\n          boolean remaining = true;\n          if (envSplit.length == 0) {\n            return\n          } else if (envSplit.length == 1) {\n            Map passage = ['\''text'\'': envSplit[0]];ctx['\''passages'\''].add(passage)\n          } else {\n            while (remaining) {\n              Map passage = ['\''text'\'': envSplit[i++]];\n              while (i < envSplit.length && passage.text.length() + envSplit[i].length() < params.model_limit) {passage.text = passage.text + '\'' '\'' + envSplit[i++]}\n              if (i == envSplit.length) {remaining = false}\n              ctx['\''passages'\''].add(passage)\n            }\n          }",
+      "params": {
+        "model_limit": 1024
+      }
+    }
+  },
+  {
+    "foreach": {
+      "field": "passages",
+      "processor": {
+        "inference": {
+          "model_id": ".elser_model_1",
+          "target_field": "_ingest._value.sparse",
+          "field_map": {
+            "_ingest._value.text": "text_field"
+          },
+          "inference_config": {
+            "text_expansion": {
+              "results_field": "tokens"
+            }
+          }
+        }
+      }
+    }
+  }
+]}'
+```	
+
+
+### Step 4: Running the fscrawler as a docker service
+
+### Step 4a. Download necessary files from the kit
+
+To get started, download the [fscrawler-docker-config.zip](./assets/fscrawler-docker-config.zip) file, extract the zip file to a directory and navigate to that directory:
+
+```
+unzip fscrawler-docker-config.zip
+cd fscrawler-docker-config
+rm ../fscrawler-docker-config.zip
+```
+
+### Step 4b. Edit the settings
+
+The `_settings.yaml` under `config/idx` is used to configure your fscrawler. Some minimal updates are needed to ensure your elasticsearch settings are updated in the `_settings.yaml`. 
+
+Run the provided `update_settings.sh` script as follows to automatically replace the placeholder variables using the values configured by you in [Step 1](#step-1-create-all-the-necessary-environment-variables).
+
+```bash
+./update_settings.sh
+```
+
+
+### Step 4c. Edit the environment file
+
+Use the provided `.env` (located at the root of `fscrawler-docker-config`) to configure which directory you would like to upload files from. 
+
+Open the file for editing and add the complete path to the documents directory you would like to use for uploading to elasticsearch, against the `DOCUMENTS_DIRECTORY` variable. An example after updating looks like this:
+
+<img src="./assets/example_volume_mount.png" width="500" height="50"/>
```

Comment:
It would be nice to provide a PDF file as an example. I tried it with the Watson Assistant docs PDF file https://cloud.ibm.com/media/docs/pdf/watson-assistant/watson-assistant.pdf, and it worked pretty well. This would also be more consistent with the other docs.